tools/runners/run-test-suite.sh

```diff
@@ -154,7 +158,8 @@ do
 
         if [ $status_code -eq 0 ]
         then
-            test $VERBOSE && echo "[$tested/$TOTAL] $cmd_line: PASS"
+            test $VERBOSE && printf "[%4d/%4d] " "$tested" "$TOTAL" &&
```

Comment:
Why is it nicer to have numbers padded to 4 characters with spaces?

Reply:
BTW, why is this not color-coded like the rest of the output?

Reply:
I padded them to four spaces because this way so that the `PASS` and `FAIL` messages always begin in the same coloumn.
It's not colored because in this `if [ "$IS_SNAPSHOT" == true ]` block we report the passing/failing of `jerry-snapshot generate` and that command can get quite long, so it's maybe more readable if it's not green color like the rest of the output. I tried it but it wasn't really better in my opinion. But I can color it if you prefer it that way.